build.sh

```diff
@@ -283,14 +283,14 @@ fi
 ######################################
 
 download_object_server() {
-    local archive_name="realm-object-server-bundled_node_darwin-$REALM_OBJECT_SERVER_VERSION.tar.gz"
-    /usr/local/bin/s3cmd get --force "s3://realm-ci-artifacts/services-bundle/$REALM_OBJECT_SERVER_VERSION/$archive_name"
+    curl -C - -L -O "http://static.realm.io/downloads/mobile-platform/v$REALM_OBJECT_SERVER_VERSION/realm-mobile-platform.zip"
     rm -rf sync
     mkdir sync
-    tar -C sync -xf $archive_name
-    rm  $archive_name
-    echo "\nenterprise:\n  skip_setup: true" >> "sync/object-server/configuration.yml"
-    touch "sync/object-server/do_not_open_browser"
+    (cd sync && unzip ../realm-mobile-platform.zip)
+    rm realm-mobile-platform.zip
+    echo "\nenterprise:\n  skip_setup: true" >> "sync/realm-object-server/object-server/configuration.yml"
+    sed -i '' -e "s/    listen_address: '0\.0\.0\.0'/    listen_address: '::'/" "sync/realm-object-server/object-server/configuration.yml"
```

Comment:
is this temporary until a new release of the object server is cut with this change baked in by default?